### PR TITLE
feat!: add casavo france

### PR DESCRIPTION
**Pull request by pikachuCa**
Add casavo france tech stack

### CODE REVIEW

File: radar/casavo-tech-radar.json [hunk 1]
```diff
@@ -530,5 +530,188 @@
...
+  },
+  {
+    "name": "Algolia",
+    "ring": "assess",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": ""
+  },
+  {
+    "name": "Argocd",
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:549]: already present

File: radar/casavo-tech-radar.json [hunk 2]
```diff
@@ -530,5 +530,188 @@
...
+  },
+  {
+    "name": "Google Calendar",
+    "ring": "adopt",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": ""
+  },
+  {
+    "name": "Grafana",
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:613]: already present

File: radar/casavo-tech-radar.json [hunk 3]
```diff
@@ -530,5 +530,188 @@
...
+    "ring": "adopt",
+    "quadrant": "tools",
+    "isNew": "TRUE",
+    "description": ""
+  },
+  {
+    "name": "Graphql",
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:620]: already present

File: radar/casavo-tech-radar.json [hunk 4]
```diff
@@ -530,5 +530,188 @@
...
+  },
+  {
+    "name": "AWS Lambda",
+    "ring": "adopt",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": ""
+  },
+  {
+    "name": "Next",
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:641]: if this is nextJs, it's already present

File: radar/casavo-tech-radar.json [hunk 5]
```diff
@@ -530,5 +530,188 @@
...
+  },
+  {
+    "name": "Play",
+    "ring": "adopt",
+    "quadrant": "languages & frameworks",
+    "isNew": "TRUE",
+    "description": ""
+  },
+  {
+    "name": "React",
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:662]: already present

File: radar/casavo-tech-radar.json [hunk 6]
```diff
@@ -530,5 +530,188 @@
...
+  },
+  {
+    "name": "Terraform",
+    "ring": "adopt",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": ""
+  },
+  {
+    "name": "Typescript",
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:704]: already present

**Comment by pikachuCa** [comment]
Duplicates were removed @edubacco

**Comment by edubacco** [comment]
Thanks @pikachuCa !
I would also add short descriptions, to understand how and why we use the blip (but not to describe the blip itself).
WDYT?

**Comment by pikachuCa** [comment]
> Thanks @pikachuCa ! I would also add short descriptions, to understand how and why we use the blip (but not to describe the blip itself). WDYT?

Done :)

File: radar/casavo-tech-radar.json [hunk 7]
```diff
@@ -530,5 +530,146 @@
...
+  },
+  {
+    "name": "Akka",
+    "ring": "adopt",
+    "quadrant": "languages & frameworks",
+    "isNew": "TRUE",
+    "description": "Mostly used for workers working on messages or large number of db rows"
+  },
+  {
+    "name": "Algolia",
```
Comment (xpepper) [inline radar/casavo-tech-radar.json:542]: @pikachuCa What id Algolia? (we could add in the description)

Comment (pikachuCa) [reply radar/casavo-tech-radar.json:542]: > To search listings on proprioo website 

A hosted version of elasticsearch

File: radar/casavo-tech-radar.json [hunk 8]
```diff
@@ -530,5 +530,146 @@
...
+    "isNew": "TRUE",
+    "description": "To store all code and scripts"
+  },
+  {
+    "name": "Google Calendar",
+    "ring": "adopt",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": "To allow brokers to manage their time"
+  },
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:604]: I think we can skip it, too detailed

File: radar/casavo-tech-radar.json [hunk 9]
```diff
@@ -530,5 +530,146 @@
...
+    "quadrant": "languages & frameworks",
+    "isNew": "TRUE",
+    "description": "Used for Mysql/Scala communication"
+  },
+  {
+    "name": "Terraform",
+    "ring": "adopt",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": "Some of the infrastructure is based on terraform (roles Mysql, eks) "
```
Comment (edubacco) [inline radar/casavo-tech-radar.json:666]: On Casavo side, the whole infrastructure is declared as a code using terraform. I would be more generic

File: radar/casavo-tech-radar.json [hunk 10]
```diff
@@ -530,5 +530,146 @@
...
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": "For production images in Casavo France"
+  },
+  {
+    "name": "AWS EKS",
+    "ring": "adopt",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": "For staging"
```
[the remark this answers is not in the document]
Comment (pikachuCa) [reply radar/casavo-tech-radar.json:581]: I prefer detailed information like "Used to orchestrate containers in AWS ecosystem on IT side and for staging only on french side."

File: radar/casavo-tech-radar.json (hunk 8 above)
Comment (pikachuCa) [reply radar/casavo-tech-radar.json:604]: I think it is a major feature for the brokers

File: radar/casavo-tech-radar.json (hunk 9 above)
Comment (pikachuCa) [reply radar/casavo-tech-radar.json:666]: I would mention the lack of terraform on some of the infrastructure actually. 
To motivate the team to bring more of the french infra to Terraform.

File: radar/casavo-tech-radar.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (edubacco) [reply radar/casavo-tech-radar.json:581]: yep, sounds good

File: radar/casavo-tech-radar.json (hunk 9 above)
Comment (edubacco) [reply radar/casavo-tech-radar.json:666]: sounds good!

File: radar/casavo-tech-radar.json [hunk 11]
```diff
@@ -530,5 +530,146 @@
...
+
+  {
+    "name": "Elasticsearch",
+    "ring": "adopt",
+    "quadrant": "tools",
+    "isNew": "TRUE",
+    "description": "For search (user feature) and logs (dev feature)"
+  },
+  {
+    "name": "Gitlab",
```
[the remark this answers is not in the document]
Comment (xpepper) [reply radar/casavo-tech-radar.json]: fixed

File: radar/casavo-tech-radar.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (xpepper) [reply radar/casavo-tech-radar.json]: fixed

**Review by xpepper — APPROVED** [review]

**Review by edubacco — APPROVED** [review]

**Pull request merged**